src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,13 @@
-export const TodoList = () => {};
+import React from 'react';
+import { TodoInfo } from '../TodoInfo';
+import { Todos } from '../../Types/Types';
+
+export const TodoList: React.FC<Todos> = ({ todos }) => {
+  return (
+    <section className="TodoList">
```

Comment:
Use semantic tags for list `ul` `li`. Don't forget to remove default `li` dots and `ul` paddings